sdk/storage/azure-storage-file-share/pom.xml

```diff
@@ -85,6 +90,24 @@
       <version>2.0-M4-groovy-2.5</version> <!-- {x-version-update;org.spockframework:spock-core;external_dependency} -->
       <scope>test</scope>
     </dependency>
+    <dependency>
+        <groupId>com.azure</groupId>
+        <artifactId>azure-storage-blob</artifactId>
+        <version>12.13.0-beta.1</version> <!-- {x-version-update;com.azure:azure-storage-blob;current} -->
+        <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-identity</artifactId>
+      <version>1.3.1</version> <!-- {x-version-update;com.azure:azure-identity;dependency} -->
+      <scope>test</scope>
+    </dependency>
+    <dependency>
```

Comment:
Do we need 2 different dependencies on blob here?

Reply:
Intellij probably added one of these. will fix